src/azul/transformer.py

```diff
@@ -300,6 +300,43 @@ def get(self):
         return thaw(super().get())
 
 
+class FrequencySetAccumulator(Accumulator):
+    """
+    An accumulator that accepts any number of values and returns a list that has
+    length max_size or smaller and is ordered by most frequent value to less frequent.
+
+    >>> a = FrequencySetAccumulator(2)
+    >>> a.accumulate('x')
+    True
+    >>> a.accumulate(['x','y'])
+    True
+    >>> a.accumulate({'x','y','z'})
+    True
+    >>> a.get()
+    ['x', 'y']
+    >>> a = FrequencySetAccumulator(0)
+    >>> a.accumulate('x')
+    True
+    >>> a.get()
+    []
+    """
+    def __init__(self, max_size) -> None:
+        super().__init__()
+        self.value = Counter()
+        self.max_size = max_size
+
+    def accumulate(self, value) -> bool:
+        original_count = sum(self.value.values())
+        if isinstance(value, (list, set)):
+            self.value.update(value)
+        else:
+            self.value[value] += 1
+        return original_count < sum(self.value.values())
```

Comment:
I don't think this can ever be false. And we don't need to return a bool since we are not bound by the SetAccumulator protocol anymore.

Reply:
Removed the return of a boolean value.